docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -954,35 +1125,35 @@ console.log(jsContext.myProductName);
 
 [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/inject.js).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=inject).
-
 ### {{jsContext}}
 
-Returns JSON for all data injected by inject helper.
+```handlebars
+{{jsContext}}
+```
+Returns JSON for all data injected by the [inject](#inject) helper.
 
 #### Example
 
-```html
-{{inject "myProductName" product.title}}
-
-<script>
-var jsContext = JSON.parse({{jsContext}});
-console.log(jsContext.myProductName);
-// => "BigCommerce Coffee Mug"
-</script>
+```handlebars
+{{need to add}}

```

Comment:
@junedkazi Can you please provide an example for this helper?

Reply:
So `jsContext` is interesting bcoz it relies on `inject` helper. All it does is whatever you inject using the inject helper before you print jsContext will get printed when you use this.

Reply:
May be we can just show a sample using the combination of `inject` and `jsContext`

Reply:
Example of inject https://github.com/bigcommerce/cornerstone/blob/0c2a8824e79f7bb9a52111558330f70d9a809593/templates/layout/base.html#L45

output will be a json representation of an object of all the keys injected

Reply:
@traci707 is working some best practices and has documented an example as well https://github.com/bigcommerce/dev-docs/pull/966/files#diff-3360974167727f4bed66304292ec08aa5dbf6a29aa1a1f3ec4698efdb0e7765aR41